test/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
This is getting injected into `document.body` so it should probably not have `<html><head>`.